src/seedu/addressbook/commands/ExitCommand.java

```diff
@@ -16,4 +16,8 @@ public CommandResult execute() {
         return new CommandResult(MESSAGE_EXIT_ACKNOWEDGEMENT);
     }
 
+    @Override
+    public boolean isMutating() {
+        return super.isMutating();
```

Comment:
Do not use super.isMutating. If you do this, why not just don't override it? An appropriate way is overriding and return true or false accordingly as per normal.